### PR TITLE
Add spdk null bdev driver

### PR DESCRIPTION
**Pull request by tiagolobocastro**
Useful for benchmarking the I/O stack with minimal overhead

### CODE REVIEW

File: spdk-sys/build.rs [hunk 1]
```diff
@@ -79,6 +79,8 @@ fn main() {
         .whitelist_function("^spdk.*")
         .whitelist_function("create_malloc_disk")
         .whitelist_function("delete_malloc_disk")
+        .whitelist_function("bdev_null_create")
+        .whitelist_function("bdev_null_delete")
```
Comment (jonathan-teh) [inline spdk-sys/build.rs:83]: The next whitelist entry should already cover these.

Comment (tiagolobocastro) [reply spdk-sys/build.rs]: done, good catch thanks :+1:

**Review by jonathan-teh — APPROVED** [review]

**Review by gila — APPROVED** [review]
Cool 😎

**Comment by gila** [comment]
Bors r+

**Pull request merged**